rpc/baremetalhostagent/agent/moc_baremetalhostagent.proto

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/baremetalhostagent";
+
+package moc.baremetalhostagent;
+import "google/protobuf/wrappers.proto";
+import "moc_common_common.proto";
+import "moc_common_computecommon.proto";
+
+
+message BareMetalHostSSHPublicKey {
+	string keydata = 1;
+}
+
+message BareMetalHostDisk {
+	string diskName = 1;
```

Comment:
What will happen is cloud agent passes in this? Is it going to format and partition the disk?

Reply:
not sure how this is relevant

Reply:
My thinking was that we will want to turn this the other way around. The bmh agent is reporting to cloud agent the configuration of the bmh so that we could potentially use this information to drive the host selection. We have been talking Vince, Davut, Chris and me about the need to split command&control messages vs reporting messages, but that has not yet happened.

Reply:
No requirement for now so I removed. We may need them for future AB update feature, we will revisit then.